Skype/WebSDK/docs/PTLocalUserNote.md

```diff
@@ -0,0 +1,24 @@
+
+# Note
```

Comment:
Maybe add a comment about what happens if you do multiple sets without waiting for completion? (order is based on what order server receives requests)